### ISSUE TITLE
Add support for latest debian stable (bookworm/12)

### ISSUE THREAD

**Issue by bsagal** [makedeb/prebuilt-mpr#172]
Please can you add support of the debian bookworm

**Comment by hwittenborn** [comment]
Yeah, not a problem at all @bsagal. I also need to add Ubuntu 23.10, so I'll just get that going while I'm at it.

**Comment by hwittenborn** [comment]
I'll try to have something going in a few days - I'm adding arm64 support in the coming days and adding both of those distros should be fairly trivial once that's going.
